benchmarks/src/main/scala/benchmark/JsonDecodeBenchmark.scala

```diff
@@ -22,77 +23,48 @@ case class SearchResp(
       currentPage: Int,
       pages: Int,
       orderBy: String,
-      results: List[JValue]
+      results: List[Json]
 )
 case class WrappedLegacyResponse(response: SearchResp)
 
 @State(Scope.Thread)
 class JsonDecodeBenchmark {
 
-  def loadJson(filename: String): (JValue, Json) = {
+  def loadJson(filename: String): Json = {
     val rawJson = Resources.toString(Resources.getResource(filename), StandardCharsets.UTF_8)
-    val jvalue = org.json4s.jackson.JsonMethods.parse(rawJson)
     val circeJson = {
       import io.circe.jawn._
       parse(rawJson).getOrElse(Json.Null)
     }
-    (jvalue, circeJson)
+    circeJson
   }
 
-  val (contentJvalue: JValue, contentJson: Json) = loadJson("content-with-blocks.json")
-  val (searchResponseJvalue, searchResponseJson) = loadJson("search-response.json")
-  val (massiveContentsListJvalue, massiveContentsListJson) = loadJson("massive-contents-list.json")
+  val contentJson = loadJson("content-with-blocks.json")
+  val massiveContentsListJson = loadJson("massive-contents-list.json")
+  val contentOpt = contentJson.as[Content].leftMap(e => throw e).toOption
```

Comment:
As this is just a benchmark, I would do a `.get` here. Seems weird to leave it as an option, and you don't want the time taken to extract the value to interfere with your benchmarking.